pkg/ccl/testccl/sqlccl/sqlccl_test: TestGCTenantJobWaitsForProtectedTimestamps failed

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8207387?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8207387?buildTab=artifacts#/) on master @ [972160e697b1c6a726a174d5c7761410996c4eac](https://github.com/cockroachdb/cockroach/commits/972160e697b1c6a726a174d5c7761410996c4eac):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3aad4e00c7d41d09dee6369c20f680d0/logTestGCTenantJobWaitsForProtectedTimestamps659738553
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:651: -- test log scope end --
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (54.42s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (47.61s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:487: condition failed to evaluate within 45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (45.61s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-23151

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8247389?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8247389?buildTab=artifacts#/) on master @ [5a42279405e6c31bc0f3ad78f96c0cc39b04e967](https://github.com/cockroachdb/cockroach/commits/5a42279405e6c31bc0f3ad78f96c0cc39b04e967):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3aad4e00c7d41d09dee6369c20f680d0/logTestGCTenantJobWaitsForProtectedTimestamps794068679
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:651: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3aad4e00c7d41d09dee6369c20f680d0/logTestGCTenantJobWaitsForProtectedTimestamps794068679
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (49.72s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (47.57s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:487: condition failed to evaluate within 45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (45.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8691945?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8691945?buildTab=artifacts#/) on master @ [8511dc08269679b6562f2010ef357c2384649871](https://github.com/cockroachdb/cockroach/commits/8511dc08269679b6562f2010ef357c2384649871):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/3aad4e00c7d41d09dee6369c20f680d0/logTestGCTenantJobWaitsForProtectedTimestamps1766902075
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:673: -- test log scope end --
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (48.22s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (47.11s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
test logs left over in: /artifacts/tmp/_tmp/3aad4e00c7d41d09dee6369c20f680d0/logTestGCTenantJobWaitsForProtectedTimestamps1766902075
    tenant_gc_test.go:502: condition failed to evaluate within 45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (45.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8750819?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8750819?buildTab=artifacts#/) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps4055996903
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:673: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps4055996903
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (54.98s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (50.47s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:502: condition failed to evaluate within 45s: expected protected timestamp record to hold up GC
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (45.28s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8758167?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8758167?buildTab=artifacts#/) on master @ [3d054f37c7c87f53cb56fac4e5500f0d1130d09a](https://github.com/cockroachdb/cockroach/commits/3d054f37c7c87f53cb56fac4e5500f0d1130d09a):


```
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.36s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:529: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3499/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:529
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3499/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:670
        	Error:      	Received unexpected error:
        	            	job 841225169794564097: could not mark as succeeded: txn exec: context canceled
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1679
        	            	  | [...repeated from below...]
        	            	Wraps: (2) job 841225169794564097: could not mark as succeeded
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:925
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:958
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:921
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:896
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1476
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1411
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:65
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:343
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:623
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1671
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1606
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:413
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:892
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (4) txn exec
        	            	Wraps: (5) context canceled
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.70s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8764692?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8764692?buildTab=artifacts#/) on master @ [e9c96e7179e19aae2f8d386f67eb950db8c3354b](https://github.com/cockroachdb/cockroach/commits/e9c96e7179e19aae2f8d386f67eb950db8c3354b):


```
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.52s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:529: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3436/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:529
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3436/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:670
        	Error:      	Received unexpected error:
        	            	job 841510593562836993: could not mark as succeeded: txn exec: context canceled
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1679
        	            	  | [...repeated from below...]
        	            	Wraps: (2) job 841510593562836993: could not mark as succeeded
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:897
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1477
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1412
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:65
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:343
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:623
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1671
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1606
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:413
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:892
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (4) txn exec
        	            	Wraps: (5) context canceled
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.55s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8920045?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8920045?buildTab=artifacts#/) on master @ [fcea283ebca17a6d923c5d4b0401697438b77dbd](https://github.com/cockroachdb/cockroach/commits/fcea283ebca17a6d923c5d4b0401697438b77dbd):


```
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:164
        	            	  | [...repeated from below...]
        	            	Wraps: (2) failed to release record 44c8fe52-c405-42fe-a0cc-36ac2342c6b6
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:845
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:425
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:476
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).ExecEx
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:689
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.storage.Release
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:160
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3.1
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:666
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1411
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1489
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1477
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1412
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:665
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (4) protectedts-Release
        	            	Wraps: (5)
        	            	Wraps: (6) failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/32/1/"D\xc8\xfeR\xc4\x05B\xfe\xa0\xcc6\xac#Bƶ",/Table/32/1/"D\xc8\xfeR\xc4\x05B\xfe\xa0\xcc6\xac#Bƶ"/PrefixEnd), [txn: d3959635], [can-forward-ts] RPC error: node unavailable; try another peer
        	            	  |
        	            	  | (opaque error leaf)
        	            	  | type name: github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/kvcoord.sendError
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *colexecerror.StorageError (6) *errbase.opaqueLeaf
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.33s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8933455?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8933455?buildTab=artifacts#/) on master @ [14b43be03c1c246765be17aa15245493a42c401d](https://github.com/cockroachdb/cockroach/commits/14b43be03c1c246765be17aa15245493a42c401d):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps83601473
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:673: -- test log scope end --
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (10.64s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': sql: database is closed
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.68s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965733?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965733?buildTab=artifacts#/) on master @ [1b162d1b274eec7b307fbbfca7294460bfdef025](https://github.com/cockroachdb/cockroach/commits/1b162d1b274eec7b307fbbfca7294460bfdef025):


```
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.storage.Release
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:164
        	            	  | [...repeated from below...]
        	            	Wraps: (2) failed to release record 201d6bd9-a62a-4cfb-87c5-c5269a803dad
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:840
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:409
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:460
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).ExecEx
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:673
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.storage.Release
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:160
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3.1
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:666
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1406
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1484
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1472
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1407
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:665
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (4) protectedts-Release
        	            	Wraps: (5)
        	            	Wraps: (6) node unavailable; try another peer
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *colexecerror.StorageError (6) *kvpb.NodeUnavailableError
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.40s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8982350?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8982350?buildTab=artifacts#/) on master @ [4dc9e98d8cc1b137ee4cc950a4a42e5895bb890b](https://github.com/cockroachdb/cockroach/commits/4dc9e98d8cc1b137ee4cc950a4a42e5895bb890b):


```
    tenant_gc_test.go:529: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2505/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:529
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2505/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:670
        	Error:      	Received unexpected error:
        	            	job 846322021952684033: could not mark as succeeded: txn exec: context canceled
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1615
        	            	  | [...repeated from below...]
        	            	Wraps: (2) job 846322021952684033: could not mark as succeeded
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1472
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1407
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:65
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:343
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:641
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1607
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1542
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:413
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:910
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (4) txn exec
        	            	Wraps: (5) context canceled
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.60s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.61s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8998222?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8998222?buildTab=artifacts#/) on master @ [01e14f68f7c757cab1c878cebfecd73c5119046b](https://github.com/cockroachdb/cockroach/commits/01e14f68f7c757cab1c878cebfecd73c5119046b):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1852129189
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:673: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1852129189
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (10.71s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.64s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:529: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2184/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:529
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2184/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/testccl/sqlccl/sqlccl_test_/sqlccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:670
        	Error:      	Received unexpected error:
        	            	job 846597216717045761: node liveness error: restarting in background
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1551
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:413
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:910
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) job 846597216717045761: node liveness error: restarting in background
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012422?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012422?buildTab=artifacts#/) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).ExecEx
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:673
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:230
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1406
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1484
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1472
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1407
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:65
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:343
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:641
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1607
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1542
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:413
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:910
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (6) job-update
        	            	Wraps: (7) failed to send RPC: sending to all replicas failed; last error: ba: Put [/Table/15/1/846886120370405377/0,/Min), Put [/Table/15/1/846886120370405377/1/1,/Min), Delete [/Table/15/2/"running"/2023-03-11T07:25:33.413831Z/846886120370405377/0,/Min), ConditionalPut [/Table/15/2/"succeeded"/2023-03-11T07:25:33.413831Z/846886120370405377/0,/Min), ConditionalPut [/Table/15/3/NULL/NULL/846886120370405377/0,/Min), Delete [/Table/15/4/"\x01\x01\x80\xd7#lTc/Fl\xb4\xfc\xb3\xad^@\xdd]"/"running"/2023-03-11T07:25:33.413831Z/846886120370405377/0,/Min), Delete [/Table/15/4/"\x01\x01\x80\xd7#lTc/Fl\xb4\xfc\xb3\xad^@\xdd]"/"running"/2023-03-11T07:25:33.413831Z/846886120370405377/2/1,/Min), [txn: 6619a644] RPC error: node unavailable; try another peer
        	            	  |
        	            	  | (opaque error leaf)
        	            	  | type name: github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/kvcoord.sendError
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *errbase.opaqueLeaf
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.53s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051326?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051326?buildTab=artifacts#/) on master @ [43f7cde6f036f720388912bde7310095c2f8ca9e](https://github.com/cockroachdb/cockroach/commits/43f7cde6f036f720388912bde7310095c2f8ca9e):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3369660325
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:673: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3369660325
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (11.17s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': dial tcp 127.0.0.1:38897: connect: connection refused
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.45s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9088983?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9088983?buildTab=artifacts#/) on master @ [d68f5cdfb25476aaf874bff0bb6515eba02efcd2](https://github.com/cockroachdb/cockroach/commits/d68f5cdfb25476aaf874bff0bb6515eba02efcd2):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3515944093
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:673: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3515944093
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (11.14s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': sql: database is closed
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107298?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107298?buildTab=artifacts#/) on master @ [98594dda15e35643e447877ee591770083bb854c](https://github.com/cockroachdb/cockroach/commits/98594dda15e35643e447877ee591770083bb854c):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps2352964365
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps2352964365
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (11.48s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': sql: database is closed
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.55s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122277?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122277?buildTab=artifacts#/) on master @ [0a72a49eb6e4ce5823b53d02125441c3da2e94f5](https://github.com/cockroachdb/cockroach/commits/0a72a49eb6e4ce5823b53d02125441c3da2e94f5):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1532036689
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1532036689
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (11.41s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.57s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:531: 
        	Error Trace:	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:531
        	            				pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:672
        	Error:      	Received unexpected error:
        	            	job 848858171093811201: node liveness error: restarting in background
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1550
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:467
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:981
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) job 848858171093811201: node liveness error: restarting in background
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.61s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122275?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122275?buildTab=artifacts#/) on master @ [0a72a49eb6e4ce5823b53d02125441c3da2e94f5](https://github.com/cockroachdb/cockroach/commits/0a72a49eb6e4ce5823b53d02125441c3da2e94f5):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps732377804
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps732377804
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (10.84s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': dial tcp 127.0.0.1:46129: connect: connection refused
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.28s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131830?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131830?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1483
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1418
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:66
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:368
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:643
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1610
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1541
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:467
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:981
        	            	  | [...repeated from below...]
        	            	Wraps: (8) failed to read query result
        	            	Wraps: (9) candidate pg code: 57014
        	            	Wraps: (10) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/util/cancelchecker.init
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/cancelchecker/cancel_checker.go:80
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6348
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.main
        	            	  | 	GOROOT/src/runtime/proc.go:233
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1594
        	            	Wraps: (11) query execution canceled
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *withstack.withStack (8) *errutil.withPrefix (9) *pgerror.withCandidateCode (10) *withstack.withStack (11) *errutil.leafError
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.80s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131826?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131826?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.InfoStorage.write
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/job_info_storage.go:126
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.InfoStorage.Write
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/job_info_storage.go:198
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.InfoStorage.WriteLegacyPayload
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/job_info_storage.go:232
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).CreateJobWithTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:651
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).CreateJobWithTxn.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:668
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).CreateJobWithTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:671
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).CreateStartableJobWithTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:806
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.TestingCreateAndStartJob.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/test_helpers.go:57
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1417
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1495
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1483
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1418
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.TestingCreateAndStartJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/test_helpers.go:56
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:661
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) write-job-info-insert
        	            	Wraps: (3) failed to send RPC: sending to all replicas failed; last error: ba: ConditionalPut [/Table/53/1/849146705697079297/"legacy_payload"/1916-10-15T16:56:29.693020999Z/0,/Min), [txn: 09d70250] RPC error: node unavailable; try another peer
        	            	  |
        	            	  | (opaque error leaf)
        	            	  | type name: github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/kvcoord.sendError
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueLeaf
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.25s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142259?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142259?buildTab=artifacts#/) on master @ [9acc75317aebfdfe69fb097a8a28e0040c7a67fd](https://github.com/cockroachdb/cockroach/commits/9acc75317aebfdfe69fb097a8a28e0040c7a67fd):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1009463940
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1009463940
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (10.97s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': dial tcp 127.0.0.1:46449: connect: connection refused
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.28s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142261?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142261?buildTab=artifacts#/) on master @ [9acc75317aebfdfe69fb097a8a28e0040c7a67fd](https://github.com/cockroachdb/cockroach/commits/9acc75317aebfdfe69fb097a8a28e0040c7a67fd):


```
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1483
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1418
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:66
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:368
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:643
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1610
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1541
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:467
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:981
        	            	  | [...repeated from below...]
        	            	Wraps: (8) failed to read query result
        	            	Wraps: (9) candidate pg code: 57014
        	            	Wraps: (10) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/util/cancelchecker.init
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/cancelchecker/cancel_checker.go:80
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6348
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6325
        	            	  | runtime.main
        	            	  | 	GOROOT/src/runtime/proc.go:233
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1594
        	            	Wraps: (11) query execution canceled
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *withstack.withStack (8) *errutil.withPrefix (9) *pgerror.withCandidateCode (10) *withstack.withStack (11) *errutil.leafError
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.80s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162875?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162875?buildTab=artifacts#/) on master @ [b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0](https://github.com/cockroachdb/cockroach/commits/b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3488911783
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3488911783
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (236.67s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (230.61s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:504: condition failed to evaluate within 3m45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (225.45s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162873?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162873?buildTab=artifacts#/) on master @ [b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0](https://github.com/cockroachdb/cockroach/commits/b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3606670578
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3606670578
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (56.96s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (51.03s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:504: condition failed to evaluate within 45s: expected protected timestamp record to hold up GC
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (46.05s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9183374?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9183374?buildTab=artifacts#/) on master @ [78bb7696847e1d8abf5d6a199c6dd3414a568691](https://github.com/cockroachdb/cockroach/commits/78bb7696847e1d8abf5d6a199c6dd3414a568691):


```
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:608
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:102
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:70
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1418
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1496
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1484
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1419
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:66
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.Update
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/update.go:368
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.Updater.succeeded
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:643
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1610
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/registry.go:1541
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/adopt.go:467
        	            	  | github.com/cockroachdb/cockroach/pkg/jobs.(*StartableJob).Start.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/jobs/jobs.go:981
        	            	  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1594
        	            	Wraps: (6) select-job
        	            	Wraps: (7)
        	            	Wraps: (8) failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/53/1/849989961819357185/"legacy_progress",/Table/53/1/849989961819357185/"legacy_progress"/PrefixEnd), [txn: 84c045db] RPC error: node unavailable; try another peer
        	            	  |
        	            	  | (opaque error leaf)
        	            	  | type name: github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/kvcoord.sendError
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *withstack.withStack (6) *errutil.withPrefix (7) *colexecerror.StorageError (8) *errbase.opaqueLeaf
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.72s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9183371?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9183371?buildTab=artifacts#/) on master @ [78bb7696847e1d8abf5d6a199c6dd3414a568691](https://github.com/cockroachdb/cockroach/commits/78bb7696847e1d8abf5d6a199c6dd3414a568691):


```
        	            	  | [...repeated from below...]
        	            	Wraps: (2) failed to release record 264aaaee-c4f6-42bf-b9a3-a040bd936da7
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:851
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:409
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:460
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).ExecEx
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:673
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.storage.Release
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:160
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3.1
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:668
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1418
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1496
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:965
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:928
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:964
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:927
        	            	  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:902
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1484
        	            	  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).Txn
        	            	  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:1419
        	            	  | pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps.func6.3
        	            	  | 	pkg/ccl/testccl/sqlccl/sqlccl_test/pkg/ccl/testccl/sqlccl/tenant_gc_test.go:667
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (4) protectedts-Release
        	            	Wraps: (5)
        	            	Wraps: (6)
        	            	Wraps: (7) failed to send RPC: sending to all replicas failed; last error: ba: Scan [/Table/31/1,/Table/31/2), [txn: 89706acc] RPC error: node unavailable; try another peer
        	            	  |
        	            	  | (opaque error leaf)
        	            	  | type name: github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/kvcoord.sendError
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *colexecerror.notInternalError (6) *colexecerror.StorageError (7) *errbase.opaqueLeaf
        	Test:       	TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (0.49s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206558?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206558?buildTab=artifacts#/) on master @ [5493fdfec4e1762c4502fb2f5d42fd28292c9c9d](https://github.com/cockroachdb/cockroach/commits/5493fdfec4e1762c4502fb2f5d42fd28292c9c9d):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps287043149
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps287043149
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (237.86s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (231.46s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:504: condition failed to evaluate within 3m45s: expected protected timestamp record to hold up GC
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (226.03s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206554?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9206554?buildTab=artifacts#/) on master @ [5493fdfec4e1762c4502fb2f5d42fd28292c9c9d](https://github.com/cockroachdb/cockroach/commits/5493fdfec4e1762c4502fb2f5d42fd28292c9c9d):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1094666048
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps1094666048
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (11.73s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': dial tcp 127.0.0.1:33675: connect: connection refused
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.33s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245126?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245126?buildTab=artifacts#/) on master @ [0fcc33bc2870961b9387999d8a9fed97fccbb2ae](https://github.com/cockroachdb/cockroach/commits/0fcc33bc2870961b9387999d8a9fed97fccbb2ae):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3934036253
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3934036253
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (237.47s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (231.08s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:504: condition failed to evaluate within 3m45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (225.92s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245123?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245123?buildTab=artifacts#/) on master @ [0fcc33bc2870961b9387999d8a9fed97fccbb2ae](https://github.com/cockroachdb/cockroach/commits/0fcc33bc2870961b9387999d8a9fed97fccbb2ae):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps4250894742
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps4250894742
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (12.14s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': dial tcp 127.0.0.1:46691: connect: connection refused
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.39s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9255034?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9255034?buildTab=artifacts#/) on master @ [d107217dac5d817cc115bc0e97b7e53c0f2878bf](https://github.com/cockroachdb/cockroach/commits/d107217dac5d817cc115bc0e97b7e53c0f2878bf):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps96432188
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps96432188
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (236.79s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (230.71s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:504: condition failed to evaluate within 3m45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (225.45s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99562 pkg/ccl/testccl/sqlccl/sqlccl_test: TestGCTenantJobWaitsForProtectedTimestamps failed [C-test-failure O-robot T-sql-queries branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9263606?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9263606?buildTab=artifacts#/) on master @ [2bd2c806ab3044569b09e0a205b5bc0452ad4e2b](https://github.com/cockroachdb/cockroach/commits/2bd2c806ab3044569b09e0a205b5bc0452ad4e2b):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3384253601
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3384253601
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (12.53s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    sql_runner.go:312: error executing 'SELECT status FROM [SHOW JOBS] WHERE description = 'GC for tenant 10'': dial tcp 127.0.0.1:46155: connect: connection refused
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (5.43s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestGCTenantJobWaitsForProtectedTimestamps [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9263607?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9263607?buildTab=artifacts#/) on master @ [2bd2c806ab3044569b09e0a205b5bc0452ad4e2b](https://github.com/cockroachdb/cockroach/commits/2bd2c806ab3044569b09e0a205b5bc0452ad4e2b):


```
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3945154947
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestGCTenantJobWaitsForProtectedTimestamps
    tenant_gc_test.go:675: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/5c751b11a9f469cbdd173f9eebde1a6d/logTestGCTenantJobWaitsForProtectedTimestamps3945154947
--- FAIL: TestGCTenantJobWaitsForProtectedTimestamps (237.02s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC
    --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC (230.41s)
=== RUN   TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time
    tenant_gc_test.go:504: condition failed to evaluate within 3m45s: load-job-query: node unavailable; try another peer
        --- FAIL: TestGCTenantJobWaitsForProtectedTimestamps/tenant_set_PTS_records_dont_affect_tenant_GC/protect_system_tenant_cluster_before_drop_time (225.10s)
```
<p>Parameters: <code>TAGS=bazel,gss,race</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGCTenantJobWaitsForProtectedTimestamps.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/testccl/sqlccl/tenant_gc_test.go
+++ b/pkg/ccl/testccl/sqlccl/tenant_gc_test.go
@@ -38,6 +38,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/cockroachdb/cockroach/pkg/util/stop"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/cockroach/pkg/util/uuid"
@@ -470,7 +471,7 @@ func TestGCTableOrIndexWaitsForProtectedTimestamps(t *testing.T) {
 func TestGCTenantJobWaitsForProtectedTimestamps(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.WithIssue(t, 94808)
+
 	defer gcjob.SetSmallMaxGCIntervalForTest()()
 
 	ctx := context.Background()
@@ -614,8 +615,11 @@ func TestGCTenantJobWaitsForProtectedTimestamps(t *testing.T) {
 		tenID := roachpb.MustMakeTenantID(10)
 		sqlDB.Exec(t, "ALTER RANGE tenants CONFIGURE ZONE USING gc.ttlseconds = 1;")
 
+		tenantStopper := stop.NewStopper()
+		defer tenantStopper.Stop(ctx) // in case the test fails prematurely.
+
 		ten, conn10 := serverutils.StartTenant(t, srv,
-			base.TestTenantArgs{TenantID: tenID, Stopper: srv.Stopper()})
+			base.TestTenantArgs{TenantID: tenID, Stopper: tenantStopper})
 		defer conn10.Close()
 
 		// Write a cluster PTS record as the tenant.
@@ -630,6 +634,10 @@ func TestGCTenantJobWaitsForProtectedTimestamps(t *testing.T) {
 			return tenPtp.WithTxn(txn).Protect(ctx, rec)
 		}))
 
+		// Ensure the secondary tenant is not running any more tasks.
+		tenantStopper.Stop(ctx)
+
+		// Drop the record.
 		sqlDB.Exec(t, `DROP TENANT [$1]`, tenID.ToUint64())
 
 		sqlDB.CheckQueryResultsRetry(

